nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/pom.xml

```diff
@@ -84,6 +84,10 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-comment-api</artifactId>
+    </dependency>
     <dependency>
```

Comment:
see my suggestion https://github.com/nuxeo/nuxeo/pull/3587/files#r366762283 we should not add this dependency `nuxeo-platform-notification-core` should not know about `nuxeo-platform-comment-api`